profiles.md

```diff
@@ -0,0 +1,234 @@
+
+Status codes:
+* 201 Created - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions
+* 409 Conflict - if for the specified eperson a profile already exists
+* 422 Unprocessable Entity - if the specified eperson does not exist
+
+In case of response with status 201 the endpoint returns the newly created resource, while in case of 409 it returns the resource that generated the conflict.
+
+To create a new profile use 
+```bash
+curl -i -X POST ${dspace-url}/api/eperson/profiles?eperson=eb645ef8-1373-41eb-bf67-6afcea7e2069 -H "Content-Type:application/json"
+```
+
+## Claim an existing Profile
+**POST /api/eperson/profiles?eperson=<:eperson-uuid>**
+
+Modify the profile item specified in the request content to make the given eperson its owner. The content-type is uri-list with the url of the item's profile to be claimed.
+This action can only be done by the EPerson themselves or an Administrator.
+
+Parameters:
+* eperson-uuid (optional): the uuid of eperson for which the profile is to be claimed; if not specified, the eperson is the authenticated one
+
+Status codes:
+* 201 Created - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions
+* 409 Conflict - if the specified eperson has already a profile or if the profile to be claimed is already owned
+* 404 Unprocessable Entity - if the specified eperson does not exist
+
+An example curl call:
+```bash
+ curl -i -X POST ${dspace-url}/api/eperson/profiles?eperson=eb645ef8-1373-41eb-bf67-6afcea7e2069 -H "Content-Type:text/uri-list" \
+ --data ${dspace-url}/api/core/items/cec6ee1b-7730-44da-a224-a7c5af63f821
+```
+
+## Hide/unhide a profile
+**PATCH /api/eperson/profiles/<:eperson-uuid>**
+
+This operation allow to change the visibility of one profile. This action can only be done by the EPerson themselves or an Administrator.
+
+To hide or unhide a profile use 
+```bash
+curl -i -X PATCH ${dspace-url}/api/eperson/profiles/eb645ef8-1373-41eb-bf67-6afcea7e2069 --data '[ { "op": "replace", "path": "/visible", "value": true }]' -H "Content-Type:application/json"
+```
+
+Status codes:
+
+* 200 OK - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions
+* 404 Not found - if a profile for the specified eperson does not exist
+
+If the operation succeed the endpoint returns the updated resource.
+
+## Delete a profile
+**DELETE /api/eperson/profiles/<:eperson-uuid>**
+
+Delete the profile related to the given eperson. This action can only be done by the EPerson themselves or an Administrator.
+The type of deletion depends on a configuration property: it can be soft, only the link is canceled, or hard, with which the item is canceled. I can express this behavior in the contract.
```

Comment:
This sentence needs some minor rewording to make it clearer.. perhaps something like this:


> The type of deletion depends on a configuration property.  It can be a soft deletion, where only the link between the EPerson and Profile object is removed (so the Profile object remains). Or, it can be a hard deletion, where the Profile object is permanently deleted.

(We might also want to specify the configuration property name in the contract eventually, if it's already known.  That way people reading the contract know what configuration to check, instead of having to search for it in DSpace docs.)